manim/animation/transform.py

```diff
@@ -227,6 +227,30 @@ def interpolate(self, alpha: float) -> None:
 
 
 class ClockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: ClockwiseTransform1
+        :quality: medium
```

Comment:
You should not specify the quality for examples in the documentation, we already render them with a predefined resolution and frame rate, see here: https://github.com/ManimCommunity/manim/blob/00f0134027496a5b4102820c59ca095caf82bfcf/manim/utils/docbuild/manim_directive.py#L198

```suggestion
```